### ISSUE TITLE
How can I package a .jar executable?

### ISSUE THREAD

**Issue by Heus-Sueh** [SalaniLeo/Immagini#8]
for example i am trying to package this launcher [here](https://github.com/ATLauncher/ATLauncher) which is only available in .jar for linux

and how do I also include [openjdk](https://jdk.java.net/19/)?

**Comment by SalaniLeo** [comment]
Yes, you can package a .jar file, but you need to create your own AppRun file. To create an apprun file you can read [the official docs](https://docs.appimage.org/packaging-guide/manual.html#manual-packaging). To include openjdk there is no way currently, but I'm working on it. Sadly a new release won't be happening for at least another 7 days because I'm busy, but after that you can expect to see a new release in a couple of days

**Comment by SalaniLeo** [comment]
I want to give you an update, I started adding more and more new feature, and at this point I already added a few new things, but I'm not ready for a release, sorry if I take long to add this feature, but I kept thinking of adding too much features and now I need more time (also now I stopped adding new things and I'm only working on implementing the new ones). I hope that by the end of the month I'll have already released a new version. Also I can tell you that the app will be translatable in the new release

**Comment by Heus-Sueh** [comment]
> I want to give you an update, I started adding more and more new feature, and at this point I already added a few new things, but I'm not ready for a release, sorry if I take long to add this feature, but I kept thinking of adding too much features and now I need more time (also now I stopped adding new things and I'm only working on implementing the new ones). I hope that by the end of the month I'll have already released a new version. Also I can tell you that the app will be translatable in the new release

no problem, go at your own pace 👍
